changelog/_template.rst

```diff
@@ -0,0 +1,40 @@
+{% for section in sections %}
+{% set underline = "-" %}
```

Comment:
I think we should use indentation in this jinja template for better readability. For HTML-like source, it's usually 2 spaces.